src/qdlutils/applications/qdlhome/main.py

```diff
@@ -0,0 +1,131 @@
+import logging
+import tkinter as tk
+
+import qdlutils.applications.qdlmove.main as qdlmove
+import qdlutils.applications.qdlple.main as qdlple
+import qdlutils.applications.qdlscan.main as qdlscan
+import qdlutils.applications.qdlscope.main as qdlscope
+
+logger = logging.getLogger(__name__)
+logging.basicConfig()
+
+class HomeApplication:
+    '''
+    This is the base application for qdlhome.
+
+    It serves as basic lanucher for each of the different applications. Note that there
+    are no special protections coded into the application to prevent user error due to 
+    improper usage (e.g. attempting to set the same DAQ output in two different apps
+    simultaneously). This is not programmed in directly since how each app handles the
+    hardware is slightly different and was not intended to be shared between multiple
+    apps at the same time. Nevertheless, the applications are generally designed not to
+    hold onto the hardware unecessarily and so interleaved calls are generally 
+    permissible --- and improper calls should generally just throw errors (although it
+    might do weird things if you're in the middle of a scan, for example).
+
+    This application is a later addition to the qdlutils package and the individual
+    apps were not initially designed to be operated in this way. Thus some bugs are
+    inevitable. Please post an issue to the github page if you encounter any unusual
+    interactions. 
+     
+    If you want to add new applications to this launcher, ensure that the structure of
+    the relevant application is appropritate (able to run as the root tk.Tk, or as a 
+    tk.Toplevel).
+
+    If you wish to include additional controls in this app specifically (such as simple
+    DAQ controls to flip optical components or toggle hardware), then you should separate
+    this script into the standard main/application_controller/application_gui structure
```

Comment:
main/application_controller/application_gui looks like a directory which could cause future confusion. Maybe say something like "... separate this script into the standard application components of main.py, application_controller.py, and application_gui.py as in the other application folders."